app/src/main/java/com/arctouch/codechallenge/util/Extensions.kt

```diff
@@ -0,0 +1,28 @@
+package com.arctouch.codechallenge.util
```

Comment:
Here you declared `Extensions` file name but it turns out that there are no Extensions functions defined. Instead, there are some utility functions.

Reply:
Yes, I have created the extensions because I would create some variations to facilitate the development, but I only added code to assist like utility functions.